crypto/jwx.go

```diff
@@ -84,6 +90,42 @@ func (client *Crypto) SignJWS(ctx context.Context, payload []byte, headers map[s
 	return signJWS(payload, headers, privateKey, detached)
 }
 
+// EncryptJWE encrypts a signed payload using the provided public key and key identifier.
```

Comment:
the payload isn't necesarrily signed? (only when used in the context of JWM)

Reply:
Correct, removed the signed word from the comment.